spec/event_store_client/client_spec.rb

```diff
@@ -47,6 +47,47 @@ module EventStoreClient
       end
     end
 
+    describe 'link_to' do
```

Comment:
Please follow the naming convention when naming describe blocks for methods: 

describe "#METHOD_NAME" - for instance mehtods
describe ".METHOD_NAME" - for class methods

look above for examples